client/src/app/auth/vuex/mutations-types.js

```diff
@@ -0,0 +1,3 @@
+// http://vuex.vuejs.org/en/mutations.html#using-constants-for-mutation-types
+export const SET_TOKEN = 'Auth//SET_TOKEN'
```

Comment:
Why is this syntax used?

Could not be?
export const SET_TOKEN = 'AUTH_SET_TOKEN' ?